ksqldb-engine/src/main/java/io/confluent/ksql/engine/QueryCleanupService.java

```diff
@@ -124,6 +126,7 @@ public QueryCleanupTask(
           .orElse(appId);
       //generate the prefix depending on if using named topologies
       this.isTransient = isTransient;
+      this.isPersistent = isPersistent;
```

Comment:
we should take another look at all the places `isTransient` is used - for example:
```java
        LOG.info("Deleting offsets for appId: {}", appId);
        tryRun(
            () -> serviceContext
                .getConsumerGroupClient()
                .deleteConsumerGroups(ImmutableSet.of(appId)),
            "internal consumer groups");
      }
```
Should we also delete offsets (consumer groups) for internal topics for persistent queries?